### PR TITLE
first draft commit

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

**Comment by zikalino** [comment]
looks huge :-)
where all the scenarios come from?

File: src/test/scenario-test.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            const modifiers = new Modifiers(await createPassThruSession(cfg, aznameryaml, 'code-model-v4'));
+            const modified = await modifiers.process();
+            const modifieryaml = serialize(modified, codeModelSchema);
+            await (writeFile(`${__dirname}/../../src/test/scenarios/${each}/modifier.yaml`, modifieryaml));
+    
+        }
+    }
+}
+
+@suite class Hello {
```
Comment (zikalino) [inline src/test/scenario-test.ts]: what's this class for? do we need it?

Comment (qiaozha) [reply src/test/scenario-test.ts]: No, we don't need it, I mean to test the scenario test in the future

**Comment by zikalino** [comment]
ok, let's take it in

**Pull request merged**